kernel/src/codec_manager.cpp

```diff
@@ -1,9 +1,17 @@
 #include "codec_manager.h"
 #include <vector>
 #include <windows.h> 
+#include <iostream>
+#include <filesystem>
+#include <list>
+#include <string>
 
 static std::vector<CodecInfo> g_codecs;
 
+std::list <std::string> recursiveDLLSearch(const std::string& search_path, const std::string& extension);
+std::string path = "bin/";
+std::string extension = ".dll";

```

Comment:
There is no need to declare global vars, if you can pass this values when calling recursiveDLLSearch

Reply:
fixed